flavors/web-app/src/main/java/cf/dmms/app/web/api/book/BookDto.java

```diff
@@ -0,0 +1,46 @@
+package cf.dmms.app.web.api.book;
+
+import java.util.List;
+
+class BookDto {
+
+	private Long id;
+	private String title;
+	private String isbn;
+	private String description;
+	private List<Long> authors;
```

Comment:
Do we want to return id's? Maybe some kind of dto?

Reply:
What DTO do you propose?

Reply:
same as Author? ID, name, surname?

Reply:
Actually Author also has list of his books. So proposed by You dto seems to be ok.